test/DynamicExpresso.UnitTest/OperatorsTest.cs

```diff
@@ -130,6 +133,34 @@ public void String_Concatenation_with_type_conversion()
 			Assert.AreEqual(1981 + "ciao ", target.Eval("1981 + \"ciao \""));
 		}
 
+        [Test]
+        public void String_Concatenation_check_string_method()
+        {
+            MethodInfo expectedMethod = typeof(string)
+                .GetMethod(nameof(String.Concat), new[] {typeof(string), typeof(string)});
+            
+            Interpreter interpreter = new Interpreter();
+
+            string expressionText = "\"ciao \" + 1981";
+
+            Lambda lambda = interpreter.Parse(expressionText);
+            
+            MethodCallExpression methodCallExpression = lambda.Expression as MethodCallExpression;
+            
+            Assert.IsNotNull(methodCallExpression);
+            Assert.AreEqual(expectedMethod, methodCallExpression.Method);
+        }
+
+        [Test]
+        public void String_Concatenation_with_null()
+        {
+            Interpreter interpreter = new Interpreter();
+
+            string expressionText = "\"ciao \" + null";
+
+            Assert.AreEqual("ciao ", interpreter.Eval(expressionText));
+        }
+
```

Comment:
What if you write:
```
        [Test]
        public void String_Concatenation_with_null()
        {
            Interpreter interpreter = new Interpreter();
            string expressionTextWithNullKeyword = "\"ciao \" + null";
            Assert.AreEqual("ciao ", interpreter.Eval(expressionTextWithNullKeyword));

            // create a function that returns null
            Func<String> someFunc = () => null;
            interpreter.SetFunction("someFunc", someFunc);
            string expressionTextWithNullFunc = "\"ciao \" + someFunc()";
            Assert.AreEqual("ciao ", interpreter.Eval(expressionTextWithNullFunc));
        }
```
My concern is that we should not call `.ToString` directly, because on null it will not work. But checking if the expression is null I think is not enough.
Maybe there is some better solution, but I need to think about it.

Also, one minor fix, sorry ;-) : can you please align code space style?
The repo is configured using tabs, I think that you have used spaces.